### PR TITLE
Virtual bump for vbox guest additions and build fixes

### PR DESCRIPTION
**Pull request by CvH**

### CODE REVIEW

File: tools/mkpkg/mkpkg_vbox-graphic [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+if [ -z "$1" ]; then
+  echo "Usage: $0 5.0.123"
+  exit 0
+fi
+
+VBOX_VERSION="$1"
+
+echo "getting sources..."
+  mkdir vbox && cd vbox/
+  wget http://download.virtualbox.org/virtualbox/5.0.16/VBoxGuestAdditions_${VBOX_VERSION}.iso
```
Comment (zalaare) [inline tools/mkpkg/mkpkg_vbox-graphic]: would http://download.virtualbox.org/virtualbox/${VBOX_VERSION}/VBoxGuestAdditions_${VBOX_VERSION}.iso be better?

Comment (CvH) [inline tools/mkpkg/mkpkg_vbox-graphic]: tx completely overlooked ;D

**Comment by lrusak** [comment]
I'm ok with this but why do you need VDPAU_SUPPORT="yes"?

**Comment by CvH** [comment]
@lrusak dropped till we have a better idea :)

**Comment by lrusak** [comment]
what about vboxguest? 

https://github.com/LibreELEC/LibreELEC.tv/blob/master/packages/linux-drivers/vboxguest/package.mk#L20

**Comment by CvH** [comment]
added fixup not sure if okay (never done it this way) - squash if okay

**Comment by lrusak** [comment]
do this 
```
diff --git a/packages/linux-drivers/vboxguest/package.mk b/packages/linux-drivers/vboxguest/package.mk
index b822b72..06b2d38 100644
--- a/packages/linux-drivers/vboxguest/package.mk
+++ b/packages/linux-drivers/vboxguest/package.mk
@@ -22,9 +22,7 @@ PKG_REV="1"
 PKG_ARCH="x86_64"
 PKG_LICENSE="GPL"
 PKG_SITE="http://www.virtualbox.org"
-PKG_URL="${DISTRO_SRC}/xf86-video-virtualbox-${PKG_VERSION}.tar.xz"
-PKG_SOURCE_DIR="xf86-video-virtualbox-${PKG_VERSION}"
-PKG_DEPENDS_TARGET="toolchain linux"
+PKG_DEPENDS_TARGET="toolchain linux xf86-video-virtualbox"
 PKG_NEED_UNPACK="$LINUX_DEPENDS"
 PKG_PRIORITY="optional"
 PKG_SECTION="driver"
@@ -37,9 +35,8 @@ PKG_AUTORECONF="no"
 pre_make_target() {
   unset LDFLAGS
 }
-
 make_target() {
-  cd src/${PKG_NAME}-${PKG_VERSION}/
+  cd $(get_build_dir xf86-video-virtualbox)/src/${PKG_NAME}-${PKG_VERSION}/
   make KERN_DIR=$(kernel_path)
 }
 
```

**Comment by CvH** [comment]
@lrusak squashed

**Comment by CvH** [comment]
@lrusak added your fixup (need to run a full rebuild to ensure that it works like desired)

**Comment by CvH** [comment]
updated to 5.0.20
gpu still not works but at least less errors
testcompile is running

**Pull request merged**